examples/Clusters/03.k8scluster-managed-namespace.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: blocks/v1beta1
+kind: K8sCluster
+metadata:
+  name: default-cluster
+spec:
+  kubeConfig:
+    secretKeyRef:
+      # Supposing we have a service account Config stored in the secrets
+      # with access to only a specific namespace
```

Comment:
We don't really care about this here. It just needs to be a kubeconfig that has admin access to the managedNamespace.